csrc/scheduler/matmul_heuristic.h

```diff
@@ -67,13 +67,13 @@ class MatmulParams : public HeuristicParams {
   //! This is the maximum vectorization supported by the inputs.
   struct SupportedVectorization {
     // operands
-    int64_t a = 8;
-    int64_t b = 8;
+    int64_t a = 16;
```

Comment:
Is this number of items or number of bytes?

Reply:
Number of items. I'll make that clear in a comment. I set it to default to the largest possible vectorization for convenience so that we can just use `std::min` to update it.

Reply:
Isn't the largest possible vectorization 8 items?
https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#data-movement-and-conversion-instructions-cp-async

Reply:
Oh, you are saying "the largest possible vectorization for all possible dtypes"?

Reply:
Yeah exactly. We could potentially have an FP8 operand at some point.